server/src/test/java/com/google/coffeehouse/storagehandler/StorageHandlerTest.java

```diff
@@ -0,0 +1,246 @@
+  }
+
+  public void insertBookWithNullAuthorAndNullIsbn() {
+    List<Mutation> mutations = new ArrayList<>();
+    mutations.add(
+      Mutation.newInsertOrUpdateBuilder("Books")
+        .set("bookId")
+        .to("bookWithNullAuthorAndNullIsbn")
+        .set("title")
+        .to("anonymous book about love")
+        .build());
+    dbClient.write(mutations);
+  }
+
+  public void insertBookWithEmptyAuthor() {
+    List<Mutation> mutations = new ArrayList<>();
+    mutations.add(
+      Mutation.newInsertOrUpdateBuilder("Books")
+        .set("bookId")
+        .to("bookWithEmptyAuthor")
+        .set("author")
+        .to("")
+        .set("isbn")
+        .to("9780060959470")
+        .set("title")
+        .to("all about love: new visions")
+        .build());
+    dbClient.write(mutations);
+  }
+
+  public void insertBookWithEmptyIsbn() {
+    List<Mutation> mutations = new ArrayList<>();
+    mutations.add(
+      Mutation.newInsertOrUpdateBuilder("Books")
+        .set("bookId")
+        .to("bookWithEmptyIsbn")
+        .set("author")
+        .to("bell hooks")
+        .set("isbn")
+        .to("")
+        .set("title")
+        .to("all about love: new visions")
+        .build());
+    dbClient.write(mutations);
+  }
+
+  public void insertBookWithEmptyAuthorAndEmptyIsbn() {
+    List<Mutation> mutations = new ArrayList<>();
+    mutations.add(
+      Mutation.newInsertOrUpdateBuilder("Books")
+        .set("bookId")
+        .to("bookWithEmptyAuthorAndEmptyIsbn")
+        .set("author")
+        .to("")
+        .set("isbn")
+        .to("")
+        .set("title")
+        .to("anonymous book about love")
+        .build());
+    dbClient.write(mutations);
+  }
+
+  public void insertBookWithEmptyAuthorAndNullIsbn() {
+    List<Mutation> mutations = new ArrayList<>();
+    mutations.add(
+      Mutation.newInsertOrUpdateBuilder("Books")
+        .set("bookId")
+        .to("bookWithEmptyAuthorAndNullIsbn")
+        .set("author")
+        .to("")
+        .set("title")
+        .to("anonymous book about love")
+        .build());
+    dbClient.write(mutations);
+  }
+
+  public void insertBookWithNullAuthorAndEmptyIsbn() {
+    List<Mutation> mutations = new ArrayList<>();
+    mutations.add(
+      Mutation.newInsertOrUpdateBuilder("Books")
+        .set("bookId")
+        .to("bookWithNullAuthorAndEmptyIsbn")
+        .set("isbn")
+        .to("")
+        .set("title")
+        .to("anonymous book about love")
+        .build());
+    dbClient.write(mutations);
+  }
+
+  @Test
+  public void addPersonClubMembershipMutation() throws Exception {
+    StorageHandlerTestHelper.insertPerson();
+    StorageHandler.getAddMembershipTransaction(dbClient, "person", "club");
+    ReadContext readContext = dbClient.singleUse();
+    Boolean actual = StorageHandlerHelper.checkMembership(readContext, "person", "club");
+    assertEquals(true, actual);
+  }
+
+  @Test
+  public void deletePersonClubMembership() throws Exception {
```

Comment:
Please modify the names of these tests to fit with our predetermined naming conventions (see https://github.com/googleinterns/step73-2020/wiki/Style-Guide#test-method-names).

Reply:
Thanks for pointing this out -- I forgot to change the name of the tests after I had changed the method names. I'll fix that now.